tox.ini

```diff
@@ -80,67 +74,51 @@ skipsdist = True
 skip_install = True
 commands = 
     poetry run python ./install_packages.py bandit
-    poetry run bandit -r aea benchmark examples packages \
```

Comment:
Replaced commands with makefile commands. 
This makes it so a) we have one version of the commands to keep track of, b) simplifies the tox file, c) keep tabs of makefile commands to check if they work.